app/controllers/prototypes_controller.rb

```diff
@@ -2,7 +2,7 @@ class PrototypesController < ApplicationController
   before_action :set_prototype, only: [:edit,:show, :update, :destroy]
 
   def index
-    @prototypes = Prototype.eager_load(:user, :main_image).order("prototypes.created_at DESC")
+    @prototypes = Prototype.eager_load(:user, :main_image).order("prototypes.created_at DESC").page(params[:page]).per(8)
```

Comment:
User Load (0.4ms)  SELECT  `users`.* FROM `users` WHERE `users`.`id` = 4 LIMIT 1
  User Load (0.4ms)  SELECT  `users`.* FROM `users` WHERE `users`.`id` = 2  ORDER BY `users`.`id` ASC LIMIT 1
  Like Load (0.3ms)  SELECT  `likes`.* FROM `likes` WHERE `likes`.`prototype_id` = 1 AND `likes`.`user_id` = 2 LIMIT 1
  Rendered prototypes/_like.html.haml (3.6ms)
  Thumbnail Load (0.4ms)  SELECT `thumbnails`.* FROM `thumbnails` WHERE `thumbnails`.`prototype_id` = 1 AND `thumbnails`.`status` = 0
  Thumbnail Load (0.5ms)  SELECT `thumbnails`.* FROM `thumbnails` WHERE `thumbnails`.`prototype_id` = 1 AND `thumbnails`.`status` = 1
   (0.3ms)  SELECT COUNT(*) FROM `comments` WHERE `comments`.`prototype_id` = 1
  Comment Load (0.3ms)  SELECT `comments`.* FROM `comments` WHERE `comments`.`prototype_id` = 1
  CACHE (0.0ms)  SELECT  `users`.* FROM `users` WHERE `users`.`id` = 4 LIMIT 1  [["id", 4]]
  User Load (0.4ms)  SELECT  `users`.* FROM `users` WHERE `users`.`id` = 2 LIMIT 1
  CACHE (0.0ms)  SELECT  `users`.* FROM `users` WHERE `users`.`id` = 2 LIMIT 1  [["id", 2]]